TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/Root.ts2

```diff
@@ -1,37 +1,41 @@
 /**
  * Root TypoScript template for the {siteName} site
+ *
+ * For more details see: http://neos.readthedocs.io/en/stable/HowTos/SelectingPageTemplate.html#using-a-defaultpage-prototype
  */
-page = TYPO3.Neos:Page {
-	head {
-		stylesheets.site = TYPO3.TypoScript:Template {
-			templatePath = 'resource://{packageKey}/Private/Templates/Page/Default.html'
-			sectionName = 'stylesheets'
-		}
 
-		javascripts.site = TYPO3.TypoScript:Template {
-			templatePath = 'resource://{packageKey}/Private/Templates/Page/Default.html'
-			sectionName = 'headScripts'
-		}
-	}
+root {
+    default {
+        type = ${q(node).property('_nodeType') + '.Document'}
```

Comment:
IMHO we could even wait until the next major after that, so people have more time to adapt. Because when we move this to the core, sites will actually start breaking after the update. Unless we can implement it in a way, that we somehow check how the `root.ts2` of the active site package is structured and only activate the matcher if it makes sense.